manifests/service.pp

```diff
@@ -44,5 +44,10 @@
       ensure => $foreman_service_ensure,
       enable => $foreman_service_enable,
     }
+
+    service { "${foreman_service}.socket":
```

Comment:
Adding management of the socket here seems to have done the trick in my upgrade testing. @ekohl is there any reason not to add this?

Reply:
Initially I left it out to remain compatible with older versions which didn't ship the `.socket` file. Back then it wasn't as relevant, now it is. I think we started to ship it in 2.1 so it deserves a note in the compatibility section that 2.0 is no longer supported (technically users could ship it themselves rather than in the RPMs, but I never went that far in documenting).

Note the **think** part. I didn't check if it was 2.1. Perhaps it was 2.0

Reply:
Checking git history, it was 2.1 that it was released as part of.

Reply:
You know me I'm all for the main branch reflecting the state of the art in the project; however, I will at least ask if this is the kind of thing that warrants a fact or some other method to conditionally wrap it based on Foreman version?